cmd/query/app/otlp_translator.go

```diff
@@ -18,7 +18,7 @@ func otlp2traces(otlpSpans []byte) ([]*model.Trace, error) {
 	if err != nil {
 		return nil, fmt.Errorf("cannot unmarshal OTLP : %w", err)
 	}
-	jaegerBatches := model2otel.ProtoFromTraces(otlpTraces)
+	jaegerBatches := jptrace.ProtoFromTraces(otlpTraces)
```

Comment:
We have a linter that prohibits accidental usage of some packages, it's worth enabling it.

Reply:
Example https://github.com/jaegertracing/jaeger/blob/e39609ec675bb4ce12524d1ede6decce44db2bc9/.golangci.yml#L128-L133